Makefile

```diff
@@ -34,7 +34,7 @@ $(LINUX):
 $(WINDOWS):
 	env CGO_ENABLED=0 GOOS=windows GOARCH=amd64 go build -o $(WINDOWS) -ldflags "-X main.date=$(DATE) -X main.vers=$(VERS) -X main.hash=$(HASH)" ./cmd/$(BIN)
 
-test: test-binary test-node test-quarkus test-go
```

Comment:
Why remove quarkus tests during build?

Reply:
@lance the test is directly executing template as it was instantiated project, this won't work for Quarkus anymore because it uses golang templating, so some pre-processing is needed. For instance java files begin with `package {{.GroupId}}.{{.ArtifactId}};` which is not valid java.